File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        result = {}
        for i,num in enumerate(nums):
            objetive = target - num
            print(objetive)
            if objetive in result:
                return [ result[objetive], i]
            else:
                result[num] = i     

        return []        

File: 0001-two-sum/0001-two-sum.ts
function twoSum(nums: number[], target: number): number[] {
  
    let result = new Map()

    let index = 0
    
    for (let num of nums){
        if (result.has(num)){
            return [result.get(num), index]
        }
        else {
            console.log(num)
            result.set(target - num,index)
            index++;
        }
    }

    return []
    

};